test/src/watcher/watcher.test.ts

```diff
@@ -377,7 +377,7 @@ describe("Watcher", () => {
         verify(mockedStore.removeById(appointment.id)).once();
     });
 
-    it("handleChanges dooes not call removeById before an appointment has expired for long enough", async () => {
+    fnIt<Watcher>(w => w.handleChanges, "dooes not call removeById before an appointment has expired for long enough", async() => {
```

Comment:
Not your typo - it was there already - but since you're editing this line could fix the `dooes`